docs/course_materials/compound_data_types.html

```diff
@@ -189,32 +244,171 @@ <h1><b>
+                        type(my_tuple) # &ltclass 'tuple'&gt
+                    </code></pre>
                 </section>
                 <section>
                     <h3><b>Creating a tuple</b></h3>
+                    <ul>
+                        <li>Create a populated tuple</li>
+                    </ul>
                     <pre><code data-trim data-noescape class="hljs lang-python">
-t = tuple()
-type(t) #  &ltclass 'tuple'&gt
+                        nums_tuple = (1, 2)
+                        print(nums_tuple) # (1, 2)
 
-t = (1, 2, 'hi')
-type(t) # &ltclass 'tuple'&gt
+                        strings_tuple = ('one', 'two')
+                        print(strings_tuple) # ('one', 'two')
 
-t = 1, 2, 'hi' # be careful
-type(t) # &ltclass 'tuple'&gt
-
-t = tuple([1,2,'yo']) # casting a list to a tuple
-print(t) # (1, 2, 'yo')
+                        mix_tuple = (1, 'two')
+                        print(mix_tuple) # (1, 'two')
+                    </code></pre>
+                </section>
+                <section>
+                    <h3><b>Tuple syntax</b></h3>
+                    <ul>
+                        <li>Technically, tuples are indicated by commas ','</li>
+                        <li>Thus, a tuple can be created like so:</li>
+                    </ul>
+                    <pre><code data-trim data-noescape class="hljs lang-python">
+                        my_tuple = 1, 2, 'three'
+                        print(my_tuple) # (1, 2, 'three')
+                        type(my_tuple) # &ltclass 'tuple'&gt
+                    </code></pre>
+                    <ul>
+                        <li>Yet, it's adviced to wrap the tuple in parentheses '()' to improve readability</li>
+                    </ul>
+                </section>
+                <section>
+                    <h3><b>Tuple unpacking</b></h3>
+                    <ul>
+                        <li>As the name suggests, the action unpacks the values of a tuple into separate variables</li>
+                    </ul>
+                    <pre><code data-trim data-noescape class="hljs lang-python">
+                        my_tuple = (1, 2, 3)
+                        a, b, c = my_tuple
+                        print(a) # 1
+                        print(b) # 2
+                        print(c) # 3
                     </code></pre>
                 </section>
                 <section data-background-size="cover" data-background-image="media/exr_background.jpeg" data-markdown="exercises/04_Compound_data_types/class_exercise_2.md" data-separator="^\n\n\n" data-separator-vertical="^\n\n">
                 </section>
             </section>
+            <!-- Slicing -->
+            <section>
+                <section>
+                    <h1>
+                        <b><span>Slicing</span></b>
+                    </h1>
+                </section>
+                <section>
+                    <h3>
+                        <b>Overview</b>
```

Comment:
Since it's an operation and not a data type, maybe change it to "What is slicing?"